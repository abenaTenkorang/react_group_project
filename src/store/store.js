import { configureStore } from '@reduxjs/toolkit';
import rocketReducer from './rocket/rocketSlice';
import missionsReducer from './missions/missions';

export default configureStore({
  reducer: {
    rockets: rocketReducer,
    missions: missionsReducer,
  },
});
