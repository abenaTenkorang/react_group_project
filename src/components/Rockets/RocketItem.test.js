import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../../store/store';
import Rocket from './RocketItem';

describe('Rocket', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Rocket />
        </Provider>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
