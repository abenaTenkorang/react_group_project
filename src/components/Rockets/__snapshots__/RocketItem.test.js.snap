// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rocket renders correctly 1`] = `
<div
  className="rocket_container"
>
  <img
    alt=""
    className="img"
  />
  <div
    className="rocket-details"
  >
    <h3
      className="name"
    />
    <p
      className="description"
    />
    <button
      className="reserve-btn"
      onClick={[Function]}
      type="button"
    >
      Reserve Rocket
    </button>
  </div>
</div>
`;
